Framework/HistogramData/inc/MantidHistogramData/HistogramRebin.h

```diff
@@ -0,0 +1,39 @@
+#ifndef MANTID_HISTOGRAMDATA_HISTOGRAMREBIN_H_
+#define MANTID_HISTOGRAMDATA_HISTOGRAMREBIN_H_
+
+#include "MantidHistogramData/DllConfig.h"
+namespace Mantid {
+namespace HistogramData {
+class Histogram;
+class BinEdges;
+
+/** HistogramRebin : TODO: DESCRIPTION
```

Comment:
Remove this line and add doxygen in the cpp file as usual (since there is no class here...)?